packages/@ngx-meta/core/README.md

```diff
@@ -328,6 +328,9 @@ export class ItemComponent implements OnInit {
     this.meta.setTitle(`Page for ${this.item.name}`);
     this.meta.setTag('og:image', this.item.imageUrl);
   }
+  ngOnDestroy() {
+     this.meta.removeTag('property="og:type"');
```

Comment:
changed readme related to tests
https://github.com/fulls1z3/ngx-meta/pull/105/files#diff-5fb66ac7cd04c7eb68a660a0348a2bf9R583